UTRA-ART/Caffeine
# CAFF-19 [incomplete]: Load and run CV model

Doesn't close CAFF-19 yet since we need to convert the segmentation mask into a usable form for virtual layers.

Adds:
- Ability to load onnx model via launch file
- Node to take an image on `/image` and then publish a binary segmentation mask on `/lane_detection_mask` 
- Preliminary instructions to install onnxruntime

Test:
- Publish an image on `/image`
- Compile and build `catkin_make && source devel/setup.bash`
- Launch the node: `roslaunch lane_detection lane_detection.launch`. Make sure to update the filepath to a valid onnx model
- Launch rviz and visualize `/lane_detection_mask`

Todo:
- Need to convert the segmentation mask into whatever virtual layer needs. @1999michael to define what he needs.

Credits to @spencerteetaert for doing most of the code, I just refactored & made the ros wrapper.

We should do the conversion in a separate PR since it's not defined yet.
